.github/workflows/build.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
         timeout-minutes: 5
         continue-on-error: true
 
-      - run: ./gradlew kotlinUpgradeYarnLock build -PredwoodNoSamples
+      - run: ./gradlew kotlinUpgradeYarnLock build ziplineApiCheck -PredwoodNoSamples
```

Comment:
Isn't this already attached to the `check` lifecycle task? I wouldn't think we need to explicitly run it.

Reply:
Confirmed that it is, and removed